lib/transport/transport-tls.c

```diff
@@ -237,6 +237,14 @@ log_transport_tls_write_method(LogTransport *s, const gpointer buf, gsize buflen
   return -1;
 }
 
+TLSSession *
```

Comment:
We should add the very least add an assertion to check if this is indeed a logtransporttls instance